src/client/src/apps/MainRoute/widgets/spotTile/epics/closePositionEpic.ts

```diff
@@ -43,19 +48,18 @@ export const closePositionEpic: ApplicationEpic = (action$, state$, { platform }
     applicationConnected,
     switchMapTo(interopSubscribe$),
     withLatestFrom(state$),
+    filter(([message, state]) => !!getSpotTilesDataFromMessage(message, state)),
     map(([message, state]) => {
-      const [msg, uuid] = message as Message
+      const [msg, uuid] = message
       const trade = createTrade(
         msg,
-        state.spotTilesData[msg.symbol],
```

Comment:
types were updated to represent the case where we might not get spotTilesData for certain currencies. So now we need to handle unhappy path with filter